static/js/typeahead_helper.js

```diff
@@ -95,6 +95,29 @@ function prefix_sort(query, objs, get_item) {
 
 }
 
+function split_by_subscribers(people) {
+    var subscribers = [];
+    var non_subscribers = [];
+    var current_stream = compose.stream_name();
+
+    if (current_stream === "") {
+        // For the home page, everyone is a subscriber.
```

Comment:
I'm a little confused should this say "For the home page" or "If there is no stream specified"?  This case covers PMs and the case that they haven't input a recipient yet.